backend/Enums/UserRole.cs

```diff
@@ -2,7 +2,7 @@
 {
     public enum UserRole
     {
-        User,
-        Admin
+        User = 0,
```

Comment:
U sure we need this shit? Cuz enums are automatically numbered

Reply:
yeee, we can skip it